staging/src/k8s.io/kubelet/pkg/apis/podresources/v1/api.proto

```diff
@@ -32,14 +32,26 @@ message AllocatableResourcesResponse {
     repeated ContainerMemory memory = 3;
 }
 
+// ListPodResourcesFilterRequest restricts the result set returned by the List API
+message ListPodResourcesFilterRequest {
```

Comment:
Just reinterating my previous [comment](https://github.com/kubernetes/kubernetes/pull/132028#discussion_r2123239726) at the corresponding API change about using a more flexible filtering mechanism (e.g., by pod phase) instead of a boolean `activeOnly` flag.